src/OpenTelemetry/Metrics/MeterProviderBuilderSdk.cs

```diff
@@ -73,7 +73,7 @@ internal MeterProviderBuilderSdk AddMeasurementProcessor(MeasurementProcessor pr
 
         internal MeterProviderBuilderSdk AddExporter(MetricProcessor processor)
         {
-            this.ExportProcessors.Add(processor);
+            this.ExportProcessors.Add(new KeyValuePair<MetricProcessor, int>(processor, this.collectionPeriodMilliseconds));
```

Comment:
I may not be seeing the whole picture here but... Does the collection period always go hand-in-hand with an exporter? Could we instead have some overloads of `AddExporter` like:

```csharp
AddExporter(MetricProcessor processor)
AddExporter(MetricProcessor processor, int collectionPeriod)
```

then maybe instead of `SetCollectionPeriod` call it `SetDefaultCollectionPeriod`?

Reply:
Absolutely! I was thinking the same thing last night, but forgot this morning! (sigh, always the case).

After talking to @jmacd regarding this issue, it seems easiest if we tied the Collection period to the same as the Export period.